PhysicsTools/PyTorch/test/testUtilities.h

```diff
@@ -0,0 +1,20 @@
+#ifndef PhysicsTools_PyTorch_test_testUtilities_h
+#define PhysicsTools_PyTorch_test_testUtilities_h
+
+#include <filesystem>
+#include <string>
+
+namespace torchtest {
+
+  std::string cmsswPath(std::string path) {
+    if (path.size() > 0 && path.substr(0, 1) != "/")
+      path = "/" + path;
+
+    std::string base = std::string(std::getenv("CMSSW_BASE"));
+    std::string rel = std::string(std::getenv("CMSSW_RELEASE_BASE"));
+    return (std::filesystem::exists(base.c_str()) ? base : rel) + path;
```

Comment:
I think the check should rather be:
  - if `$CMSSW_BASE/path` exists, use that;
  - otherwise, if `$CMSSW_RELEASE_BASE` is set and `$CMSSW_RELEASE_BASE/path` exists, use that;
  - otherwise, fail.

The current check would pick the wrong path if `$CMSSW_BASE` exists (which is very likely) but `$CMSSW_BASE/path` does not (which is possible if _that_ package was not checked out).

Having said that, I think it would be simpler to use the existing `edm::FileInPath` functionality ?

Reply:
@makortel does the framework provide (or could provide) this functionality ?
It's not exactly like `FileInPath`, but there are some overlaps.